src/main/java/Duke.java

```diff
@@ -1,10 +1,151 @@
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
 public class Duke {
+    static String face = "/(*´∇｀*)/";
```

Comment:
Similarly, these emojis could be differentiated, such as ELATED_FACE and RELAXED_FACE

Reply:
It is not recommended to use numbers to distinguish between variables (Under Naming in Code Quality)